en/config_heli/README.md

```diff
@@ -0,0 +1,87 @@
+# Helicopter Configuration
+
+This section contains topics related to helicopter configuration and tuning.
+
+:::note
+Helicopters are less well-supported than other types of vehicles.
+We would welcome your [contribution](../contribute/README.md) of new features, new frame configurations, or other improvements.
+:::
+
+## Supported Configurations
+
+Supported helicopter configurations:
+- Single main rotor with any type of swash-plate
+- ESC tail rotor
```

Comment:
What is an ESC tail rotor? i.e. given there is no configuration of this rotor does this just mean you attach an ESC + motor to the tail and everything the system will magically work out everything else it needs to do?